src/System.Device.Gpio/System/Device/Spi/Devices/Windows10SpiDevice.cs

```diff
@@ -23,9 +24,14 @@ internal class Windows10SpiDevice : SpiDevice
         /// </param>
         public Windows10SpiDevice(SpiConnectionSettings settings)
         {
-            if (settings.DataFlow != DataFlow.MsbFirst || settings.ChipSelectLineActiveState != PinValue.Low)
+            if (settings.ChipSelectLineActiveState != PinValue.Low)
             {
-                throw new PlatformNotSupportedException($"Changing {nameof(settings.DataFlow)} or {nameof(settings.ChipSelectLineActiveState)} options is not supported on the current platform.");
+                throw new PlatformNotSupportedException($"Changing{nameof(settings.ChipSelectLineActiveState)} options is not supported on the current platform.");
+            }
+
+            if (settings.DataFlow == DataFlow.MsbFirst)
+            {
+                _isInverted = true;
```

Comment:
I think this should be `== DataFlow.LsbFirst`, previously we used to throw for Lsb and now we will invert